frameworks/JavaScript/hapi/start-servers.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PORT=3000 node create-server.js &
+PORT=3001 node create-server.js &
+PORT=3002 node create-server.js &
+PORT=3003 node create-server.js &
+PORT=3004 node create-server.js &
+PORT=3005 node create-server.js &
+PORT=3006 node create-server.js &
+PORT=3007 node create-server.js &
```

Comment:
Is the intention here to spin up 8 cores because that's what you were testing on? To scale correctly and test in other environments, could you do this dynamically? https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Dart/stream/start-servers.sh as an example. You can build the nginx conf the same way.

Reply:
Changing now. Will run a verify